### ISSUE TITLE
add new tutorials

### ISSUE THREAD

**Issue by minotogna** [openforis/fra-platform#3297]
@pengchenglai (cc @yaguzmang ) regarding #3285  

The current tutorials are uploaded on Youtbute
@pengchenglai Please copy the url in a comment (of this issue), after uploading the tutorials on Youtube.

The task #3285  will be closed without the new tutorials

**Comment by minotogna** [comment]
**Note: New two tutorials Translations**

> EN
> --
> The FRA platform user tutorial - How to add a reference link for a national data point
> The FRA platform user tutorial - How to add a reference link for national data
> 
> AR
> --
> البرنامج التعليمي لمنصة الفرا - كيفية إضافة رابط مرجعي لنقطة البيانات الوطنية؟
> البرنامج التعليمي لمنصة الفرا - كيفية إضافة رابط مرجعي للبيانات الوطنية؟
> 
> 
> ES
> --
> Tutorial del usuario de la plataforma FRA -  Cómo agregar un enlace de referencia para un punto de datos nacional
> Tutorial del usuario de la plataforma FRA - Cómo agregar un enlace de referencia para datos nacionales
> 
> RU
> --
> Руководство пользователя платформы ОЛР - Как добавить ссылку для национальной точки данных
> Руководство пользователя платформы ОЛР - Как добавить ссылку на национальные данные
> 
> ZH
> --
> 全球森林资源评估平台用户教程 - 如何为国家数据点添加参考链接
> 全球森林资源评估平台用户教程 - 如何为国家数据添加参考链接
> 
> FR
> --
> Tutoriel pour l'utilisateur de la plateforme FRA -  Comment ajouter le lien d'une référence pour un point de données national
> Tutoriel pour l'utilisateur de la plateforme FRA -  Comment ajouter le lien d'une référence pour des données nationales

**Comment by minotogna** [comment]
@pengchenglai any news on the video links?

**Comment by pengchenglai** [comment]
@minotogna 

I sent the request but did not receive the feedback. Now there is a update for adding the link to the data source, so that the video needs to be updated. I will check with the admins if there are any other videos that need to be made. Let's hold this for now.

**Comment by minotogna** [comment]
Thanks for the update @pengchenglai

**Comment by minotogna** [comment]
@pengchenglai should we keep this issue still active ?

**Comment by pengchenglai** [comment]
@minotogna 

Yes, a request was sent last week I will talk to Anssi today if possible.

**Comment by minotogna** [comment]
@pengchenglai I guess I can close this issue ?

**Comment by pengchenglai** [comment]
@minotogna 

Yes, plz.
